### ISSUE TITLE
Unable to Fork File from Angular

### ISSUE THREAD

**Issue by julietteeasley** [stackblitz/webcontainer-core#569]
### Describe the bug

Unable to make any changes to the files or fork. 
Taken from angular website (https://material.angular.io/cdk/drag-drop/examples#:~:text=Drag%26Drop%20enter-,predicate,-link) 

### Link to the blitz that caused the error

https://stackblitz.com/run?file=src/app/cdk-drag-drop-enter-predicate-example.ts

### Steps to reproduce

1. Try and make any changes (wont reload)
2. Try and fork (errors in console)
Error message: 
jsonWorker.js:7  Fetch API cannot load stackblitz:/node_modules/%40angular/cli/lib/config/schema.json. URL scheme "stackblitz" is not supported.

### Expected behavior

Page should rebuild on changes/save

### Parity with Local

- [ ] I have run the project in my local machine and I could not reproduce the issue.

### Platform

```
Browser name  = Chrome 
Full version  = 97.0.4692.99
Major version = 97
navigator.appName = Netscape
navigator.userAgent = Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/97.0.4692.99 Safari/537.36
performance.memory = {
  "totalJSHeapSize": 131638475,
  "usedJSHeapSize": 127070931,
  "jsHeapSizeLimit": 4294705152
}
Hash = 5df97b5e61f923919ea927c406a4806b792de8c6
```

**Comment by jrvidal** [comment]
@julietteeasley Thanks for the report.

The link you provided (https://stackblitz.com/run?file=src/app/cdk-drag-drop-enter-predicate-example.ts) is meant for a POST request, we can't find the project by following it. Could you share the original link you clicked in the Angular tutorial?

**Comment by julietteeasley** [comment]
Angular just has a button to “edit in stackblitz” with no actual URL. Here’s a link to the part of the page that the link to the example stackblitz is on:  https://material.angular.io/cdk/drag-drop/examples#cdk-drag-drop-enter-predicate

**Comment by pavankjadda** [comment]
I have the same issue. @jrvidal Is there a way to disable web containers?

**Comment by crisbeto** [comment]
I can reproduce it as well. Here's a step-by-step:

1. Go to any page on material.angular.io. I'll use https://material.angular.io/components/button/overview as an example.
2. Press the "Open in Stackblitz" button on any of the examples.
3. Once the project has been set up, press the "Fork" button.
4. Observe how there's a 500 error on the POST request to https://stackblitz.com/api/projects.

For reference, here's how the Stackblitz is created https://github.com/angular/material.angular.io/blob/master/src/app/shared/stack-blitz/stack-blitz-writer.ts#L92.

**Comment by devversion** [comment]
We figured out what was going on in the Angular Material StackBlitz examples. We are using the StackBlitz SDK and delivered a binary file `favicon.ico` to the StackBlitz endpoint. The example worked as expected in the browser then, but just upon saving/forking, nothing happened and a `Error 500` response appeared (see https://github.com/stackblitz/webcontainer-core/issues/552 for details).

A fix for the Angular Material docs site has been merged: https://github.com/angular/material.angular.io/pull/1117. We will re-deploy that one very soon. I think this issue could be closed, though the fact that many people submitted issues here on the StackBlitz side shows that some better error handling would be nice to have (helping reducing issues here :P)
